### PR TITLE
Added thecut, vulture, and grubstreet

### PR DESCRIPTION
**Pull request by CalvinWu4**
Go to an article on [thecut.com](https://www.thecut.com/2020/10/biden-harris-campaign-covid-19.html), [vulture.com](https://www.vulture.com/2020/10/the-liberating-beauty-of-the-final-destination-movies.html), and [grubstreet.com](https://www.grubstreet.com/2020/10/koreatown-nyc-adapts-to-covid-regulations.html).
Notice that the paywall loads after a while and you can no longer scroll on the page. Also notice that article content is deleted.

### CODE REVIEW

File: src/js/background.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ const allowCookies = [
   'ed.nl',
   'examiner.com.au',
   'ft.com',
+  'grubstreet.com',
+  'www.grubstreet.com',
```
Comment (iamadamdev) [inline src/js/background.js:23]: Is this needed?

File: src/js/background.js [hunk 2]
```diff
@@ -45,13 +47,17 @@ const allowCookies = [
...
+  'www.thecut.com',
   'thediplomat.com',
   'themercury.com.au',
   'thestar.com',
   'towardsdatascience.com',
   'trouw.nl',
   'vn.nl',
   'volkskrant.nl',
+  'vulture.com',
+  'www.vulture.com',
```
Comment (iamadamdev) [inline src/js/background.js:60]: It doesn't work without this? and the rest of www.*

Comment (CalvinWu4) [reply src/js/background.js]: Ok I removed the www. urls

**Comment by CalvinWu4** [comment]
Also I have a version that saves the article content and adds it back after it is removed by the paywall
This way would preserve the user's sign in cookies but the user would see the article content disappear and reappear
Which method would you prefer?

**Comment by iamadamdev** [comment]
> Also I have a version that saves the article content and adds it back after it is removed by the paywall
> This way would preserve the user's sign in cookies but the user would see the article content disappear and reappear
> Which method would you prefer?

Current method is fine

**Pull request merged**